jib-maven-plugin/README.md

```diff
@@ -285,6 +286,25 @@ In this configuration, the image:
 </configuration>
 ```
 
+In this configuration, the image:
+* Is built from a base of `gcr.io/distroless/java:debug` to include busybox
+* Sets an entrypoint to run a script `/bin/start.sh` that is provided in `src/main/jib/bin/start.sh`
+
+```xml
+<configuration>
+  <from>
+    <image>gcr.io/distroless/java:debug</image>
+  </from>
+  <container>
+    <!-- Maven doesn't check the names of child tags --> 
+    <entrypoint>
+      <executable>/busybox/sh</executable>
+      <script>/bin/start.sh</script>
```

Comment:
I couldn't understand what it means by "Maven doesn't check the names of child tags" until I saw examples using `<arg>` below. This also gives me the impression that `<executable>` and `<script>` are fixed names that I must use. TBH, I think this is confusing. It also makes me to believe that the order of `<executable>` and `<script>` shouldn't matter. I'd just go with using a single tag name.